<map id="sysio::chain_plugin::handle_guard_exception" name="sysio::chain_plugin::handle_guard_exception">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,35,175,75"/>
<area shape="rect" id="Node000002" href="$namespaceappbase.html#a99374dc23934f008a09f2f79be9eae40" title=" " alt="" coords="243,13,342,38"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="175,40,228,32,228,38,175,45"/>
<area shape="rect" id="Node000004" href="$classappbase_1_1application.html#a81aa58d3f94570b7976145039155fc46" title=" " alt="" coords="223,63,363,103"/>
<area shape="poly" id="edge3_Node000001_Node000004" title=" " alt="" coords="175,64,207,69,206,74,175,69"/>
<area shape="rect" id="Node000003" href="$classappbase_1_1application.html#ac874b2659b400fe254f1ae4260d06ecf" title=" " alt="" coords="411,5,551,46"/>
<area shape="poly" id="edge2_Node000002_Node000003" title=" " alt="" coords="343,23,395,23,395,28,343,28"/>
</map>
